### ISSUE TITLE
Nextcloud 15 compatibitity

### ISSUE THREAD

**Issue by violoncelloch** [splitt3r/mindmaps#68]
Nextcloud 15 has been released today. It would be nice if you could update this app to work on Nextcloud 15! Thank you!

**Comment by splitt3r** [comment]
You are right. But i currently have no time and the app needs a lot of refactoring. I would love to see a community contribution for that.
